src/blindrsa.ts

```diff
@@ -188,6 +188,17 @@ export class BlindRSA {
     ): Promise<Uint8Array> {
         const algorithmName = privateKey.algorithm.name;
         privateKey.algorithm.name = BlindRSA.NATIVE_SUPPORT_NAME;
+        // only creates a copy if the private key algorithm name cannot be overwritten
```

Comment:
Additionally to pwu's comments: note that the [spec establishes](https://www.w3.org/TR/WebCryptoAPI/#dfn-CryptoKey) that the fields of a key are readonly.

```js
interface CryptoKey {
  readonly attribute [KeyType];
  readonly attribute boolean [extractable];
  readonly attribute object [algorithm];
  readonly attribute object [usages];
};
```